chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -96,7 +105,52 @@
           <device>/dev/<%= @boot_device %></device>
         <% end %>
         <initialize config:type="boolean">true</initialize>
-        <partitions config:type="list"/>
+        <% if @target_platform_version.to_f <= 12 %>
+          <partitions config:type="list">
+            <partition>
+              <create config:type="boolean">true</create>
+              <filesystem config:type="symbol">ext3</filesystem>
```

Comment:
do we need to hardcode the filesytem? I would have expected that this is just the empty list like before.